template/.github/PULL_REQUEST_TEMPLATE/pre-release.md

```diff
@@ -0,0 +1,21 @@
+## Bump Rust Dependencies for Stackable Release XX.(X)X
+
+<!--
+    Replace 'TRACKING_ISSUE' with the applicable release tracking issue number.
+-->
+
+Part of <https://github.com/stackabletech/issues/TRACKING_ISSUE>
+
+> [!NOTE]
+> During a Stackable release we need to update various Rust dependencies before
+> entering the final release period to ensure we run the latest versions of
+> crates. These bumps also include previously updated and released crates from
+> the `operator-rs` repository.
+
+```[tasklist]
+### Tasks
+- [ ] Bump `stackable-operator` and friends.
+- [ ] Bump `product-version`.
+- [ ] Bump all other dependencies.
```

Comment:
Shall we mention how? And including the nix deps?

```suggestion
- [ ] Bump all other dependencies with `cargo update` and `make regenerate-nix`.
```

Or leave it to a help section at the bottom (which could also mention bumping minimum versions in `cargo.toml`?

Reply:
I think a more thorough how-to / help section might be better here.

Reply:
Mentioned in b4974c5